docs/sdks/server-sdks/node/intro.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: SDK Guide
+sidebar_position: 1
+---
+import FeatureCard from '/src/components/FeatureCard';
+
+The Eppo Javascript SDK allows you to manage feature flags and experiments in your Javascript applications.
```

Comment:
~Javascript~ -> Node.js

Actually, I think this whole page needs to be updated for Node 😅